etc/kayobe/pulp.yml

```diff
@@ -369,6 +369,16 @@ stackhpc_pulp_repository_rpm_repos: >-
   {%- endfor -%}
   {{ rpm_repos }}
 
+# Mellanox OFED version
+stackhpc_pulp_mlnx_ofed_version: 24.04-0.7.0.0
+
+# Mellanox OFED repositories
+stackhpc_pulp_repo_mlnx_ofed:
+  name: Mellanox Technologies mlnx_ofed {{ stackhpc_pulp_mlnx_ofed_version }}
+  url: "{{ stackhpc_release_pulp_content_url }}/mlnx_ofed/{{ stackhpc_pulp_mlnx_ofed_version }}/rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}/x86_64/{{ stackhpc_pulp_repo_mlnx_ofed_version }}"
+  distribution_name: "mlnx_ofed_{{ stackhpc_pulp_mlnx_ofed_version }}-"
+  base_path: "mlnx_ofed/{{ stackhpc_pulp_mlnx_ofed_version }}/rhel9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}/x86_64/"
+
```

Comment:
This should move to a separate section, just before containers on L426.

Reply:
Moved to MellanoxOFED section